src/pinns_pde_solve.jl

```diff
@@ -1406,6 +1439,25 @@ function discretize_inner_functions(pde_system::PDESystem, discretization::Physi
             end
             nothing
         end
+    if adaloss isa InverseDirichletAdaptiveLoss
+        weight_change_inertia = discretization.adaptive_loss.weight_change_inertia
+        function run_loss_inverse_dirichlet_adaptive_loss(0)
+            if iteration[1] % adaloss.reweight_every == 0
+                pde_grads_std_all = [std(Zygote.gradient(pde_loss_function, 0)[1]) for pde_loss_function in pde_loss_function]
+                pde_grads_std_max = maximum(pde_grads_std_all)
+                bc_grads_std = [std(Zygote.gradient(bc_loss_function, 0)[1]) for bc_loss_function in bc_loss_funcitons]
+
+                nonzero_divisor_eps =  adaloss_T isa Float64 ? Float64(1e-11) : convert(adaloss_T, 1e-7)
```

Comment:
What's the justification on the `1e-11` and `1e-7`?

Reply:
Bump on this.

Reply:
I looked over the implementation by the authors of the paper and they don't include an additional epsilon in the denominator, so I think I should remove this.